formats/role.jsonnet

```diff
@@ -0,0 +1,85 @@
+(import "shared/default_format.jsonnet") + {
+  definitions: (import "shared/definitions/_whitehall.jsonnet") + {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        permanent_secretary: {
+          type: "boolean",
+        },
+        cabinet_member: {
+          type: "boolean",
+        },
+        slug: {
+          type: "string",
+          description: "eg: 'prime-minister'.",
+        },
+        chief_of_the_defence_staff: {
```

Comment:
I've got a feeling that this isn't actually used anywhere.  The current CDS is not flagged as `true` for this and neither is anyone else.  I also don't think it's exposed in the UI anywhere.